src/components/views/styl/Metric.styl

```diff
@@ -57,20 +60,52 @@ canvas {
 }
 
 #metric-details {
+    margin-bottom: 20px;
+
     dl {
-        display: flex;
+        display: grid;
         font-size: .8em;
+        font-size: 12px;
+        grid-gap: $legend-grid-gap;
+        grid-template-columns: 160px 160px 160px;
         margin: 0;
+        max-width: $chart-grid-width;
 
         div {
             display: flex;
-            margin-left: 10px;
+            position: relative;
 
-            &:first-child {
-                margin-left: 0;
+            &:hover dd {
+                display: block;
             }
         }
     }
+    dt {
+        border-radius: 3px;
+        color: #fff;
+        cursor: pointer;
```

Comment:
Maybe we should keep this off until we have the click event working?

Reply:
Yea was initially convinced that would make it in this patch.